docs/README.md

```diff
@@ -39,6 +39,12 @@ installed. Also install the following libraries:
 $ sudo gem install jekyll jekyll-redirect-from rouge
 ```
 
+If your ruby version is 3.0 or higher, you should install `webrick` too.
```

Comment:
Shall we omit `too` here because the other modules are not the same here?

Reply:
@AngersZhuuuu, can you address this? You could just say like "you should also install `webrick`"

Reply:
> @AngersZhuuuu, can you address this? You could just say like "you should also install `webrick`"

Just wake up, updated.